apis/core-api/src/core/services/UserService.ts

```diff
@@ -76,16 +76,21 @@ class UserService extends BaseService {
   static async findByUniqueIdentifier(
     identifier: UserUniqueIdentifierInterface
   ): ServiceResultType<null | User> {
-    if (
-      !identifier.email &&
-      !identifier.phone &&
-      !identifier.username &&
-      !identifier.id
-    ) {
+    const query = this.queryInit();
```

Comment:
Please make sure your code is understandable, let's write simple comments as a brief notes for the line or block

Reply:
okay, will add notes